interface/patient_file/report/patient_report.php

```diff
@@ -1,10 +1,35 @@
 <?php
+/**
+ *
+ * Comprehensive Patient Report
+ *
+ * Copyright (C) 2016 OpenEMR
```

Comment:
Try interface/main/main_screen.php as an example. Note the original license was GPL2 so we should respect that.